src/RobotLoader/RobotLoader.php

```diff
@@ -347,43 +344,92 @@ public function setAutoRefresh($on = TRUE)
 
 
 	/**
+	 * Sets path to temporary directory.
 	 * @return self
 	 */
-	public function setCacheStorage(Nette\Caching\IStorage $storage)
+	public function setTempDirectory($path)
 	{
-		$this->cacheStorage = $storage;
+		$this->tempDirectory = $path;
 		return $this;
 	}
 
 
 	/**
-	 * @return Nette\Caching\IStorage
+	 * @return void
 	 */
-	public function getCacheStorage()
+	private function loadCache()
 	{
-		return $this->cacheStorage;
+		if (!$this->tempDirectory) {
+			return;
+		}
+		$file = $this->getCacheFile();
+		$list = @include $file;
```

Comment:
maybe assign directly to `$this->classes`?
that would allow omitting `$list = $this->classes` and `$this->classes = $list` lines